internal/testrunner/reporters/outputs/file.go

```diff
@@ -56,3 +56,12 @@ func reportToFile(pkg, report string, format testrunner.TestReportFormat) error
 
 	return nil
 }
+
+// testReportsDir returns the location of the directory to store test reports.
+func testReportsDir() (string, error) {
```

Comment:
Might be a side issue - 

Is this directory cleaned up before a next run or does all test results accumulate in a single place?

Reply:
Good question. I'm pretty sure the test results will just accumulate. I'm not sure if that's desired behavior or not. I can see both sides of the argument:
- it's good to let them accumulate because then we have results from previous runs around (and maybe `elastic-package clean` should clean them out), _or_
- we should clean up each time so only the latest results are available at any time.

Obviously, in a CI context this doesn't matter as we will have a fresh environment each time.

Unless you have a preference one way or the other, for now I'm okay with letting them accumulate. If we get feedback about this causing issues, we can address it then. WDYT?

Reply:
My thoughts on letting tests accumulate. I haven't seeing this behavior in my past experience in the Java world, where opinionated build systems such as Maven/Gradle define the life cycle. If the report-to-file option is enabled, defaulting to false using standard output, they simply refresh test reports on each test run, redirecting the standard output to the file (I always take a look at how Maven does the thing: http://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#redirectTestOutputToFile).

If we do this in another way, I think it could lead to unexpected reports from previous executions in the local machine, although, as you said, in CI this shouldn't be a problem. But I'd prefer having a standard best practice, where the "oracles" (or the behaviours contributors expect -internal and external) are clear.

So I'd go with both approaches: `elastic-package clean` should clean the test reports, and the test command should start a new file.

Reply:
Thanks for these insights, @mdelapenya — much appreciated!

Accordingly, I will update this PR to make the test command clean the test folder so it writes completely new files. We have a separate issue to implement `elastic-package clean`; I have made a note there: https://github.com/elastic/elastic-package/issues/13#issuecomment-722174887.

Reply:
Happy to help 😃